workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx

```diff
@@ -30,8 +34,33 @@ const ManagementToolbarResultsBar: React.FC<ManagementToolbarResultsBarProps> =
 		dispatch({payload: null, type: ListViewTypes.SET_CLEAR});
 	};
 
+	const handleRemoveItemFromFilter = (itemToRemove: string) => {
+		const searchParams = new URLSearchParams(location.search);
+
+		const filtro = searchParams.get('filter');
+
+		if (filtro) {
+			const filtroJSON = JSON.parse(decodeURIComponent(filtro));
+
+			delete filtroJSON[itemToRemove];
+
+			if (!Object.keys(filtroJSON).length) {
```

Comment:
Pay attention the word needs to be in English

```suggestion
		if (filter) {
			const filterJSON = JSON.parse(decodeURIComponent(filter));

			delete filterJSON[itemToRemove];

			if (!Object.keys(filterJSON).length) {
```